x-pack/legacy/plugins/ml/public/jobs/new_job_new/pages/components/pick_fields_step/components/split_cards/split_cards.tsx

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { FC, memo, Fragment } from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiPanel, EuiHorizontalRule, EuiSpacer } from '@elastic/eui';
+
+import { SplitField } from '../../../../../../../../common/types/fields';
+import { JOB_TYPE } from '../../../../../common/job_creator/util/constants';
+
+interface Props {
+  fieldValues: string[];
+  splitField: SplitField;
+  numberOfDetectors: number;
+  children: JSX.Element;
+  jobType: JOB_TYPE;
+}
+
+interface Panel {
+  panel: HTMLDivElement;
+  marginBottom: number;
+}
+
+export const SplitCards: FC<Props> = memo(
+  ({ fieldValues, splitField, children, numberOfDetectors, jobType }) => {
+    const panels: Panel[] = [];
+
+    function storePanels(panel: HTMLDivElement | null, marginBottom: number) {
+      if (panel !== null) {
+        panels.push({ panel, marginBottom });
+      }
+    }
+
+    function getBackPanels() {
+      panels.length = 0;
+
+      const fieldValuesCopy = [...fieldValues];
+      fieldValuesCopy.shift();
+
+      let margin = 5;
+      const sideMargins = fieldValuesCopy.map((f, i) => (margin += 10 - i)).reverse();
+
+      setTimeout(() => {
+        panels.forEach(p => (p.panel.style.marginBottom = `${p.marginBottom}px`));
+      }, 100);
+
+      const SPACING = 100;
+      const SPLIT_HEIGHT_MULTIPLIER = 1.6;
+      return fieldValuesCopy.map((fieldName, i) => {
+        const diff = (i + 1) * (SPLIT_HEIGHT_MULTIPLIER * (10 / fieldValuesCopy.length));
+        const marginBottom = -SPACING + diff;
+
+        const sideMargin = sideMargins[i];
+
+        const style = {
+          height: `${SPACING}px`,
+          marginBottom: `-${SPACING}px`,
+          marginLeft: `${sideMargin}px`,
+          marginRight: `${sideMargin}px`,
+          transition: 'margin 0.2s',
```

Comment:
does this need cross-browser alternatives since it's not running through SCSS compilation?

Reply:
Good point, I think it should probably be moved to SCSS.
I'll look into that in a follow up https://github.com/elastic/kibana/projects/30#card-23885080

Reply:
Here is how the cards split on IE11:

![image](https://user-images.githubusercontent.com/7405507/61118598-f8ec6a80-a490-11e9-8257-d2529c94b8fc.png)